server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -614,8 +613,9 @@ public void update(LiveTServerSet current, Set<TServerInstance> deleted,
         log.error("{}", e.getMessage(), e);
       }
 
-      tStop = System.currentTimeMillis();
-      log.info(String.format("Collect cycle took %.2f seconds", ((tStop - tStart) / 1000.0)));
+      final long tStop = System.nanoTime();
+      log.info(String.format("Collect cycle took %.2f seconds",
+          (TimeUnit.NANOSECONDS.toMillis(tStop - tStart) / 1000.0)));
```

Comment:
Shouldn't this just convert directly to seconds instead of millis only to divide by 1000?

Reply:
I wanted to preserve the fractional seconds.  Using TimeUnits.X.toSeconds() would just truncate.  The other option could be to use 100ths (or something) and round, but the existing log statements were of the form xxx.## - which seemed a reasonable thing to keep .

Reply:
Okay, good point. Thousandths is probably fine.